src/modules/service-provider/repo/service-provider.repo.ts

```diff
@@ -129,4 +138,20 @@ export class ServiceProviderRepo {
 
         return mapEntityToAggregate(serviceProviderEntity);
     }
+
+    public async fetchRolleServiceProvidersWithoutPerson(
```

Comment:
A repository should not return Entities. It returns Aggregates.
I think you need to use RolleRepo to load the Rolle Aggregates instead of using this method.